### PR TITLE
update doc contribution info

### PR DESCRIPTION
**Pull request by nabdelgadir**
Connect to https://github.com/strongloop/loopback-next/issues/1853.

Some updates to the contributing to docs information.

### CODE REVIEW

File: pages/en/contrib/pages.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Under English, the sub-folders are:
 - `contrib` - Information on contributing to the project, published at [http://loopback.io/doc/en/contrib/](http://loopback.io/doc/en/contrib/).
 - `community` - Area where community members can put information about their LoopBack-related projects.
 
+However, for LoopBack 4 (`lb4`), the documentation is sourced from the `/docs/site` folder in the `loopback-next` repository.
```
Comment (b-admike) [inline pages/en/contrib/pages.md]: Is the `lb4` needed here? Feel free to keep it, but I think it's better to remove it.

**Review by b-admike — APPROVED** [review]

File: pages/en/contrib/Including-READMEs.md [hunk 2]
```diff
@@ -28,3 +27,5 @@ There are two Bash scripts:
 - [`update-community-readmes.sh`](https://github.com/strongloop/loopback.io/blob/gh-pages/update-community-readmes.sh) - Gets community READMEs and puts them into `pages/en/community/readmes`.
 
 We have a [Jenkins pipeline](https://github.com/strongloop/loopback.io/blob/gh-pages/Jenkinsfile) that runs the scripts whenever there is a commit to the loopback.io repository.  
+
+As for READMEs from `loopback-next`, they are always fetched together with other doc files. To add a new README to the docs, just create a new page with [README layout](https://loopback.io/doc/en/contrib/readmes.html) in the `loopback-next` repository.
```
Comment (bajtos) [inline pages/en/contrib/Including-READMEs.md]: Can you add an example please - a link pointing to one of LB4 pages with README layout? For example: https://github.com/strongloop/loopback-next/blob/master/docs/site/Self-hosted-REST-API-Explorer.md

File: pages/en/contrib/doc-contrib.md [hunk 3]
```diff
@@ -10,10 +10,11 @@ summary: We welcome contributions to LoopBack documentation!
...
+LoopBack documentation is sourced in both the [strongloop/loopback.io](https://github.com/strongloop/loopback.io/) and [strongloop/loopback-next](https://github.com/strongloop/loopback-next/) GitHub repositories, and this site is generated using Jekyll and GitHub pages.  Some pages are copied from the `loopback-next` repository's `master` branch into `loopback.io`'s `gh-pages` branch, where this site is published from.  Don't use the `master` branch in the `loopback.io` repository.
 
-All the pages are in the `pages` directory.  Every page also has an **Edit this page** button at the bottom
+When contributing documentation to the `loopback-next` repository, a new version of `@loopback/docs` must be published to `npm` before the changes appear live. There is a script that runs automatically at 3pm EST to propogate the changes to the `loopback.io` site.
+
+All the pages from the `loopback.io` repo are in the `pages` directory and the pages from the `loopback-next` are in the `docs/site` directory.  Every page also has an **Edit this page** button at the bottom
```
Comment (bajtos) [inline pages/en/contrib/doc-contrib.md]: > the pages from the `loopback-next` are in the `docs/site` directory.

If find this little bit confusing.

The pages in `loopback-next` are stored in `docs/site` directory, but then they are copied to `pages/en/lb4` in `loopback.io`.

Could you please clarify this in your text?

File: pages/en/contrib/pages.md (hunk 1 above)
Comment (bajtos) [inline pages/en/contrib/pages.md]: Would you mind replacing `loopback-next` with a link pointing to GitHub repo?

For example:

```markdown
[`loopback-next`](https://github.com/strongloop/loopback-next)
```

The directory name `/docs/site` could be replaced with a link too.

File: pages/en/contrib/sidebar_navigation.md [hunk 4]
```diff
@@ -27,6 +27,10 @@ Sidebars are defined in YAML files in the `_data/sidebars` directory.  They are
 | community_sidebar.yml | For "Community projects" |
 | xx_lb2_sidebar.yml | Localized sidebar for language "xx" (e.g. "es" for Spanish).
 
+{% include note.html content="
+`lb4_sidebar.yml` is sourced from [loopback-next](http://github.com/strongloop/loopback-next/)
```
Comment (bajtos) [inline pages/en/contrib/sidebar_navigation.md]: Please consider adding link to the actual `lb4_sidebar.yml` file: https://github.com/strongloop/loopback-next/blob/master/docs/site/sidebars/lb4_sidebar.yml

**Review by bajtos — APPROVED** [review]
Looks mostly good 👍

**Review by jannyHou — APPROVED** [review]
👍 LGTM

**Pull request merged**